docs/usage/python.md

```diff
@@ -74,12 +82,15 @@ e.g.:
 
 <!-- prettier-ignore -->
 !!! tip
-    The index-url found in the `requirements.txt` file takes precedence over a `registryUrl` configured like the above.
-    To override the URL found in `requirements.txt`, you need to configure it in `packageRules`, as they are applied _after_ package file extraction.
+    If a `requirements.txt` file has a index-url then Renovate follows that link, instead of following any link set in the `registryUrls` array.
+    To override the URL found in `requirements.txt`, you must create a custom `packageRules` setting.
+    This is because `packageRules` are applied _after_ package file extraction.
+    OPTION 1: Create new secion in private packages docs, and add this link: Go to the [private package support docs](https://docs.renovatebot.com/getting-started/private-packages/#pip).
+    OPTION 2: Link to https://docs.renovatebot.com/python/#specify-url-in-configuration
```

Comment:
I found a section on the "Python package manager support" page [^specify-url-in-configuration] which explains how you can set a `registryUrls` array:

> ### Specify URL in configuration
>
> You can use the `registryUrls` array to configure alternate index URL(s). e.g.:
>
> ```json
> {
>  "python": {
>    "registryUrls": ["http://example.com/private-pypi/"]
>  }
> }
> ```

But the `registryUrl` config option docs [^registry-url-config-option] has this example:

> ```json
> {
>   "packageRules": [
>     {
>       "matchDatasources": ["docker"],
>       "registryUrls": ["https://docker.mycompany.domain"]
>     }
>   ]
> }
> ```

One example uses a `"python"` object, the other uses a `matchDatasources` config option. I don't know which one is correct when you have you want to override the `index-url` entry in your `requirements.txt` file... 😄 

[^specify-url-in-configuration]: https://docs.renovatebot.com/python/#specify-url-in-configuration
[^registry-url-config-option]: https://docs.renovatebot.com/configuration-options/#registryurls